spec/dor/permissable_spec.rb

```diff
@@ -0,0 +1,307 @@
+require 'spec_helper'
```

Comment:
These specs implement the requirements in
https://consul.stanford.edu/display/chimera/Repository+Roles+and+Permissions